part1/src/index.js

```diff
@@ -19,31 +27,31 @@ const App = () => {
       </div>
     )
   }
-  
+
   const Part = (props) => {
     return (
       <div>
         <p>
           {props.part} {props.exercises}
         </p>
-        </div>
+      </div>
     )
   }
 
   const Content = () => {
     return (
       <div>
-       <Part part={part1} exercises={exercises1}/>
-       <Part part={part2} exercises={exercises2}/>
-       <Part part={part3} exercises={exercises3}/>
+        <Part part={part1.name} exercises={part1.exercises} />
+        <Part part={part2.name} exercises={part2.exercises} />
+        <Part part={part3.name} exercises={part3.exercises} />
```

Comment:
Es nezinu cik daudz tu zini par ES6 fīčām kā [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) un [spreading](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax)...

Bet, ja tu `Part` komponenti taisītu, lai izmanto `name` kā `name`, nevis pārsauc par `part`, tad tu varētu [šeit izmantot spread sintaksi un padot propus React'am tādā veidā](https://reactjs.org/docs/jsx-in-depth.html#spread-attributes), reducējot kodu:

`<Part {...part1} />` kas, pārtulkotu kodu par `<Part name={part1.name} exercises={part1.exercises} />`.

Reply:
Paldies! Es uz šī esmu uzsēdusies nākamajā uzdevumā. Vēlos atbrīvoties no tiem lielajiem drukas darbiem un prātoju kā. 

(Atzīšu gan, ka šodien man būs vēlāks sākums - nav tā, ka es visu nedēļas nogali par to domāju. Paliku pie tā piektdienas vakarā.)